src/commands/GetListCommand.ts

```diff
@@ -16,14 +22,32 @@ export class GetListCommand extends Command {
 			return 1;
 		}
 
+		switch (this.shell) {
+			case undefined:
+			case 'bash':
+			case 'fish':
+				break;
+			default:
+				this.context.stderr.write('Please specify a known shell: bash, fish\n');
+				return 1;
+		}
+
 		try {
 			const projectNames = [];
 			for (const root of roots) {
 				const dirContents = await fs.readdir(root.path, { withFileTypes: true });
-				const prefix = `${root.name}${namespaceSeparator ?? ':'}`;
-				projectNames.push(
-					...dirContents.filter(entry => entry.isDirectory()).map(entry => `${prefix}${entry.name}`)
-				);
+				const dirs = dirContents.filter(entry => entry.isDirectory());
+				switch (this.shell) {
+					case 'bash':
+						projectNames.push(...dirs.map(entry => entry.name));
```

Comment:
IMO the list should contain both prefixed and unprefixed entries for bash (quoting myself, it should contain unprefixed entries _additionally_).

Also, after the roots loop, there may have to be some deduplication, in case multiple roots have a project with the same name, but I don't know how bash handles duplicates, so it might as well be unnecessary.

Reply:
Oops, missed that somehow, sounds reasonable

Reply:
Looks like bash will include the entry in the completion list multiple times, removing duplicates seems like a good thing.

Reply:
... nevermind. `compgen` won't do filtering, but readline will when actually displaying to the user. It also sorts alphabetically by default, which seems like something that should probably be left on. (Was originally thinking about putting the unqualified names last)